### DISCUSSION TITLE
How to adjust the size of the coordinate axis font?

### DISCUSSION THREAD

**Discussion by smallrain1996** [neperfepx/neper#282] [Q&A] [answered]
```
$ neper -T -n 100 -morpho "columnar(x)" -domain "cube(400,400,400)"
$ neper -V n100-id1.tess -datacellcol id -datacsyscoo -80:-80:-80 -showcsys 1 -datacsyslength 80 -datacsysrad 2 -print img
```

The above is a very simple command to model the side length not 1 but a larger number like 400?
We can adjust the length and radius of the axes so that the axes are visible, but X1 X2 X3 are not visible because they are too small.

Obviously I could add X1 X2 X3 myself using Photoshop. but I think Neper is the coolest software on my computer so I was hoping to know how to set this font size?

**Comment by rquey** [comment, marked as answer]
Actually, there is option `-datacsysfontsize` (default value `1`), but it was not documented - it is now (version `4.2.1-8`).

**Reply by smallrain1996** [reply]
Thank you very much for your reply！
